frontend/src/core/comments/components/AddComment.js

```diff
@@ -56,7 +56,13 @@ export default function AddComments(props: Props) {
                 return null;
             }
 
-            addComment(comment, translationId);
+            if (translation) {
+                addComment(comment, translation);
+            }
+            else {
+                addComment(comment);
+            }
```

Comment:
Not anymore. My original setup did not always pass a value for `translation` but with the changes it will either be a number or null, so this is no longer needed 😊